main/views.py

```diff
@@ -43,3 +43,13 @@ def form_valid(self, form: CustomUserCreationForm) -> HttpResponse:
         if form.is_valid():
             form.save()
         return super().form_valid(form)
+
+
+def about(request: HttpRequest) -> HttpResponse:
```

Comment:
I think we should start using generic, class-based views everywhere here. So changing this to be a class that uses a [generic `TemplateView`](https://docs.djangoproject.com/en/5.2/ref/class-based-views/base/#django.views.generic.base.TemplateView) is the best option here.